keyboards/hackpad/info.json

```
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "Nico Stuhlmueller",
+    "keyboard_name": "hackpad",
+    "maintainer": "ThePurox",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgblight": true,
+        "encoder" : true
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B3", "B2", "B6"],
+        "rows": ["C6", "D7", "E6", "B4", "B5"]
+    },
+    "encoder" : {
+        "rotary" : [
+            {"pin_a" : "F5", "pin_b" : "F4"}
+        ]
+    },
+    "processor": "atmega32u4",
+    "url": "https://git.imaginaerraum.de/Purox/Hackpad",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "rgblight" : {
+        "led_count" : 16,
+        "pin": "D4",
+        "hue_steps" : 20,
+        "brightness_steps" : 10,
+        "saturation_steps" : 20,
+        "max_brightness" : 255,
+        "sleep" : true,
+        "animations": {
+            "breathing" : true,
+            "rainbow_mood" : true,
+            "rainbow_swirl" : true,
+            "snake"   : true,
+            "knight"  : true,
+            "christmas" : true,
+            "static_gradient" : true,
+            "rgb_test" : true,
+            "alternating" : true,
+            "twinkle" : true
+        },
+    },
+    "layouts": {
+        "LAYOUT_numpad_4x4": {
```

```suggestion
        "LAYOUT": {
```
`LAYOUT_numpad_4x4` looks [like this](https://github.com/qmk/qmk_firmware/blob/d3b8179f467da23fec98e97b28684c43e9e8ba73/layouts/default/numpad_4x4/default_numpad_4x4/keymap.c), which does not match your board.  There does not seem to be any well-known layout that matches your board, so the layout macro name should be just `LAYOUT`; the rest of code needs to be changed accordingly at the same time.